profile/2020-09-01-hybrid.json

```diff
@@ -0,0 +1,624 @@
+        },
+        {
+          "resourceType": "locations",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "locations/quotas",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "locations/skus",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "checkNameAvailability",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "operations",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "operationResults",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "IotHubs",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "backupProviders",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        },
+        {
+          "resourceType": "backupProviders/operationResults",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/iothub/resource-manager/Microsoft.Devices/preview/2019-07-01-preview/iothub.json"
+        }
+      ]
+    },
+    "microsoft.eventhubs": {
+      "2018-01-01-preview": [
+        {
+          "resourceType": "availableClusterRegions",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/EventHub-preview.json"
+        },
+        {
+          "resourceType": "clusters",
+          "path":"https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/EventHub-preview.json"
+        },
+        {
+          "resourceType": "locations",
```

Comment:
Can you confirm the path for it? There is no such file or similar file for `loation` under `2018-01-01-preview`. Infact, all the files under `eventhubs` is sepreated.

Reply:
lookint at it

Reply:
removed the locations operations and updated teh files